internal/collection/entry/entry.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package entry
```

Comment:
i wanted to avoid the usage of the `collection` package which is now deprecated from a different migration in the future and this folder structure does not prevent that.

Reply:
The problem is they can technically still import the package if they use the correct path.

Also the current code structure, the commands need the repair package. So we anyway needed to import 
`github.com/ethersphere/bee-repair/migrations/himalaya/internal` in the `cmd` package, which is why I thought it doesn't make sense to have multiple internal packages for each migration.